flink-connector-mongodb-cdc/src/main/java/com/ververica/cdc/connectors/mongodb/internal/MongoDBConnectorSourceTask.java

```diff
@@ -102,7 +108,13 @@ public void commitRecord(SourceRecord record, RecordMetadata metadata)
             if (sourceRecords != null && !sourceRecords.isEmpty()) {
                 outSourceRecords = new LinkedList<>();
                 for (SourceRecord current : sourceRecords) {
+                    if (isHeartbeatRecord(current)) {
+                        outSourceRecords.add(markHeartbeatRecord(current));
+                        continue;
+                    }
+
                     markRecordTimestamp(current);
```

Comment:
how about rename method name to `markTimestampForHeartbeatRecord` and  `markTimestampForDataRecord` ?
It's better If we can abstract a new method `markTimestampForRecord` and do all mark works in that method will make the code more readable.